src/bindings/python/tests/test_onnx/utils/model_importer.py

```diff
@@ -28,9 +29,10 @@
     Sequence,
 )
 
-
-# add post-processing function as part of test data
-ExtOnnxTestCase = namedtuple("TestCaseExt", OnnxTestCase._fields + ("post_processing",))
+if getattr(OnnxTestCase, '_fields', None):
+    ExtOnnxTestCase = OnnxTestCase._fields + ("post_processing",)
+else:  # for ONNX >= 1.12
+    ExtOnnxTestCase = tuple((field.name for field in dataclasses.fields(OnnxTestCase))) + ("post_processing",)
```

Comment:
In ONNX < 1.12 `OnnxTestCase` is a `namedtuple`, in ONNX == 1.12 it's a `dataclass`.